traitsui/qt4/toolkit.py

```diff
@@ -26,10 +26,17 @@
 _app = pyface_toolkit("init:_app")
 
 from traits.trait_notifiers import set_ui_handler
+from pyface.api import SystemMetrics
 from pyface.qt import QtCore, QtGui, qt_api
 
 from traitsui.toolkit import Toolkit
-from .constants import screen_dx, screen_dy
+
+
+#: Screen width
+screen_dx = SystemMetrics()._get_screen_width()
```

Comment:
This is the getter for a property, and so shouldn't be called.  Just access `.screen_width`.
```suggestion
screen_dx = SystemMetrics().screen_width
```